include/devicetree/dma.h

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @brief DMA Devicetree macro public API header file.
+ */
+
+/*
+ * Copyright (c) 2020, Linaro Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DEVICETREE_DMAS_H_
+#define ZEPHYR_INCLUDE_DEVICETREE_DMAS_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup devicetree-dmas Devicetree dma API
+ * @{
+ */
+
+/**
+ * @brief Get dma controller "name" (label property) by name
+ *
+ * It's an error if the dma controller referenced by the phandle
+ * in property "dmas" at index "idx" has no label property.
```

Comment:
"idx" doesn't seem to be an argument to this macro. Are there some copy/paste issues in this docstring?

Reply:
Yes, there are. I haven't paid much attention to the doc part yet, wanted to get initial feedbacks first.
Will update.